codegenerator/cli/templates/static/codegen/src/eventFetching/FetchState.res

```diff
@@ -172,16 +159,17 @@ let mergeIntoPartition = (p: partition, ~target: partition, ~maxAddrInPartition)
           status: {
             fetchingStateId: None,
           },
-          kind: Normal({
-            contractAddressMapping: mergedContractAddressMapping,
-            dynamicContracts: mergedDynamicContracts,
-          }),
+          selection: target.selection,
+          contractAddressMapping: mergedContractAddressMapping,
+          dynamicContracts: mergedDynamicContracts,
           fetchedEventQueue: mergeSortedEventList(p.fetchedEventQueue, target.fetchedEventQueue),
           latestFetchedBlock,
         },
         rest,
       )
     }
+  | ({selection: Wildcard(_)}, _)
+  | (_, {selection: Wildcard(_)}) => (p, Some(target))
```

Comment:
So basically merge always fails with a wildcard partition?

Is there any problematic cases where it could - say - continuously try and merge partitions that can't be merged?

Reply:
No, there's no case when it creates a merge query for a partition.